### ISSUE TITLE
Adapt functional tests to Loki parameters

### ISSUE THREAD

**Issue by Doy-lee** [loki-project/loki#591]
Currently uses Monero addresses and binary names and so forth.

**Comment by josephnicholas** [comment]
@Doy-lee will work on this issue.

**Comment by josephnicholas** [comment]
@Doy-lee some rpc-fields are not yet available in the current loki code. Should I directly merge them from Monero?and example is `wide_cumulative_difficulty`.

**Comment by jagerman** [comment]
No; `wide_cumulative_difficulty` is deliberately not merged (and probably won't be) because it is a pain to deal with, because we are nowhere even remotely close to the limits of a 64-bit int, and because there are better ways to handle it (for example, resetting it and/or rescaling difficulty values at some fork height).
